extern/bundled/python-virtualenv/install.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail -x
+
+unset PYTHONPATH  # existing PYTHONPATH can interfere
+
+( # install a Python virtualenv
+if ! type virtualenv &>/dev/null; then
+    # if virtualenv isn't available, install it locally along with pip
+    [[ -e bootstrap/virtualenv.py ]] || {
+        curl -RLO https://bootstrap.pypa.io/get-pip.py &&
+        python get-pip.py virtualenv --ignore-installed --target bootstrap
```

Comment:
I'd also like to default to python3 but need to check if virtualenv is standalone or needs a host python3.  ddlib provides some crucial decorators for UDFs (`@tsv_extractor` parsing/formatting PG TSV and `@tsj_extractor` for TSJ in the future) and the new automatic feature lib is not here yet, so yes users want/need ddlib.